docs/examples/revocation/1. Create Credential definition.md

```diff
@@ -0,0 +1,96 @@
+# Revocation
+
+## Why revocation
+
+Revocation enables an issuer to revoke credentials i.e. make a credential
+invalid some time after issuance.
+The process to make credentials revokable is quite straight forward, but there
+is a few things the issuer should keep track of.
+The goal of these docs are to explain everything involved in revocation.
+
+## Creating the credential definition
```

Comment:
This should all be merged with the existing sections creating credential definitions